train.py

```diff
@@ -119,15 +106,27 @@ def train_gpu(rank, world_size, opt, trainset, trainset_temporal):
             dataloader_test = None
             dataloader_test_temporal = None
 
+    rank_0 = rank == 0
+    opt.total_iters = 0  # the total number of training iterations
+    if opt.output_display_env == "":
+        opt.output_display_env = opt.name
+
+    if rank_0:
+        visualizer = Visualizer(
+            opt
+        )  # create a visualizer that display/save images and plots
+
+        if opt.train_continue:
+            opt.train_epoch_count = visualizer.load_data()
+            opt.total_iters = opt.train_epoch_count * trainset_size
```

Comment:
I think this is correct but the name is very misleading. `train_epoch_count` should be `train_current_epoch` and `total_iters` should be `current_iter`

Reply:
Unfortunately both variable names are historical, not feeling like I want to modify this in this PR. Here `total_iters` is set into `opt` for passing it down to CM, but the variable predates this.